include/nlohmann/detail/input/input_adapters.hpp

```diff
@@ -147,6 +146,7 @@ class input_buffer_adapter : public input_adapter_protocol
     {
         if (JSON_HEDLEY_LIKELY(cursor < limit))
         {
+            assert(cursor != nullptr && limit != nullptr);
```

Comment:
This library uses alternative operator keywords. Please change to:

```suggestion
            assert(cursor != nullptr and limit != nullptr);
```

Reply:
@nlohmann Done. That's something I have to get used to.